src/main/java/util/PrototypeUtils.java

```diff
@@ -93,7 +94,8 @@ public static void setupPrototypes(){
                 SeriesData.class.getName(),
                 Graph.class.getName(),
                 XYSeries.class.getName(),
-                BubbleSeries.class.getName()
+                BubbleSeries.class.getName(),
+                SendAction.class.getName(),
```

Comment:
Whoops, this looks like a bug on the commcare side actually, too, if didn't add this.

Reply:
Ha, sure enough! https://github.com/dimagi/commcare-android/pull/1851